src/components/lists/TodosList.vue

```diff
@@ -410,12 +421,44 @@ export default {
       return (
         Math.floor((this.timeEstimated ? this.timeEstimated : 0) / 60 / 8) <= 1
       )
+    },
+
+    isEpisodeVisible() {
+      return (
+        this.displayedTasks.filter(
+          task => task.source_id || task.episode_names?.length > 0
+        ).length > 0
+      )
```

Comment:
Prefer to use `some` instead of `filter`:

```suggestion
      return this.displayedTasks.some(
        task => task.source_id || task.episode_names?.length > 0
      )
```